toast/toast.css

```diff
@@ -0,0 +1,20 @@
+.toast {
+  width: 18rem;
+  border: 2px solid #bb4430;
+  padding: 0.5rem 1rem;
+  display: flex;
+  flex-wrap: wrap;
+  word-break: break-all;
+  align-items: center;
+  justify-content: space-between;
+  border-radius: 5px;
```

Comment:
use relative unit for more responsive

Reply:
Yaa.. we should use rem.. But this was for border.. so i given it in px

Reply:
> Yaa.. we should use rem.. But this was for border.. so i given it in px

ok fine